src/kontrol/prove.py

```diff
@@ -480,6 +480,10 @@ def _method_to_cfg(
         init_node = cfg.create_node(init_cterm)
         new_node_ids = [init_node.id]
         init_node_id = init_node.id
+        if not method.is_testfail and not method.is_test:
```

Comment:
It should be sufficient to just check if the method starts with `test`, since `test` is a prefix of `testFail` and a function starting from  `testFail` returns `true` for both `is_testfail` and `is_test`:
```suggestion
        if not method.is_test:
```

Reply:
Makes sense! Addressed, thank you!